src/components/KeyboardAwareScrollView/index.tsx

```diff
@@ -67,18 +68,25 @@ type KeyboardAwareScrollViewProps = {
 const KeyboardAwareScrollView: FC<KeyboardAwareScrollViewProps> = ({
   children,
   bottomOffset = 0,
+  resetScroll = true,
   ...rest
 }) => {
   const scrollViewAnimatedRef = useAnimatedRef<Reanimated.ScrollView>();
   const scrollPosition = useSharedValue(0);
   const position = useSharedValue(0);
   const keyboardHeight = useSharedValue(0);
+  const keyboardWillAppear = useSharedValue(false);
   const tag = useSharedValue(-1);
   const initialKeyboardSize = useSharedValue(0);
   const scrollBeforeKeyboardMovement = useSharedValue(0);
+  const resetScrollPosition = useSharedValue(resetScroll);
   const { input } = useReanimatedFocusedInput();
   const layout = useSharedValue<FocusedInputLayoutChangedEvent | null>(null);
 
+  useEffect(() => {
+    resetScrollPosition.value = resetScroll;
```

Comment:
the prop changes don't get picked up by the worklets

Reply:
@bcgilliom Even if we add this property to dependency array of worklet hooks, i. e. to `useKeyboardHandler`?

Reply:
ohh, I might have missed that - will try 👍

Reply:
ok, that's weird - I swear I tried that, but seems to be working - update incoming :-)